app/move/controllers/create/base.js

```diff
@@ -10,11 +10,25 @@ class CreateBaseController extends FormWizardController {
 
   middlewareLocals() {
     super.middlewareLocals()
+    this.use(this.setButtonText)
     this.use(this.setCancelUrl)
     this.use(this.setMoveSummary)
     this.use(this.setJourneyTimer)
   }
 
+  setButtonText(req, res, next) {
+    const nextStep = this.getNextStep(req, res)
+    const steps = Object.keys(req.form.options.steps)
+    const lastStep = steps[steps.length - 1]
+    const buttonText = nextStep.includes(lastStep)
+      ? 'actions::schedule_move'
+      : 'actions::continue'
+
+    req.form.options.buttonText = req.form.options.buttonText || buttonText
```

Comment:
In which cases does `buttonText` already exist on `options`?

Reply:
At moment, none in this journey. It was more of a forethought for some of the search steps. We might want the ability to have the button text say `Search` instead of `Continue`.